test/runAnalysis/runAnalysisADD.py

```diff
@@ -0,0 +1,443 @@
+import argparse, subprocess, os	
```

Comment:
Why do we need a new file for this? The new functionality should just be added to the existing runAnalysis.py